sdk/communication/azure-communication-common/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.16.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.17.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
What is unreleased_com?

Reply:
according to @alzimmermsft in [this PR](https://github.com/Azure/azure-sdk-for-java-pr/pull/1324): _This is a special tag for using a source code dependency and will block this library from accidentally releasing if it is being used_